src/niscope/templates/_library_interpreter.py/fetch_array_measurement.py.mako

```diff
@@ -0,0 +1,2 @@
+<%page args="f, config, method_template"/>\
```

Comment:
Note: I think I need this template because I needed one for the `_grpc_stub_interpreter.py` to avoid the extra method there. Here, though, we should just forward to the default_method.py.mako template.